Tracking/test/runTracksFromGenParticles.py

```diff
@@ -0,0 +1,52 @@
+from Configurables import ApplicationMgr
+from Configurables import EventCounter
+from Gaudi.Configuration import INFO, WARNING, DEBUG
+import os
+
+if not os.path.isfile("ddsim_output_edm4hep.root"):
+    os.system("ddsim --enableGun --gun.distribution uniform --gun.energy '10*GeV' --gun.particle e- --numberOfEvents 100 --outputFile ddsim_output_edm4hep.root --random.enableEventSeed --random.seed 42 --compactFile $K4GEO/FCCee/IDEA/compact/IDEA_o1_v02/IDEA_o1_v02.xml")
```

Comment:
Hi Juan, thanks for the comments! Not sure I understand this one though. This line is just for convenience (and for tests). So that I can run/test the genTracking just calling one script